manifests/rule.pp

```diff
@@ -142,7 +148,7 @@
     $filename = "${ferm::configdirectory}/chains/${table}-${chain}.conf"
   }
 
-  $rule = squeeze("${comment_real} ${proto_real} ${proto_options_real} ${dport_real} ${sport_real} ${daddr_real} ${saddr_real} ${action_real};", ' ')
+  $rule = squeeze("${comment_real} ${proto_real} ${proto_options_real} ${outerface_real} ${dport_real} ${sport_real} ${daddr_real} ${saddr_real} ${action_real};", ' ')
   if $ensure == 'present' {
     if $interface {
```

Comment:
We've some logic here to put interface specific rules in a custom block. I suggest to repeat this pattern for outerface as well.

Reply:
I'm not sure I'd know how to do that and handle rules that could have both an interface and outerface e.g a Forward rule.

Is there a particular reason why the interface is being placed in a block and not just directly in a rule?

I've used it in ferm directly if I want to put multiple rules in with the same matching criteria but I can't see here why it's being used, unless I'm missing a trick (probably am)

I actually have a fork where I removed that logic because in the case of wireguard interfaces it seemed to be placing the rules outside the block, that said I did that to get my specific use case working as a quick and dirty fix.

I'm rather far out of my comfort zone as I'm not a developer and I don't know ruby so it's probably just a case of me not understanding fully what that logic is doing or some of the cases where it's used.